lib/template-dependencies.js

```diff
@@ -82,6 +70,29 @@ ${formattedError}`,
     .catch(handleInternetAccessError);
 }
 
+function formatElmJsonError(error, options) {
+  if (error.stderr === undefined) {
+    return error.message;
+  }
+
+  const stderrMessage = error.stderr.toString().trim();
+  const exec = /^([^]+\n)?--\s([A-Z ]+)\s-*\n\n([^]+)$/.exec(stderrMessage);
```

Comment:
I tweaked the regex to support stuff being printed before the `-- HEADER -----` as well, as in:

```
phase: retrieve
 Jan 22 18:15:41.748 WARN Failed to fetch versions from package.elm-lang.org

-- NO VALID PACKAGE VERSION ----------------------------------------------------

Because elm/json does not appear to exist and this project depends on elm/json
at any version, no valid set of package versions could be found.
```

Reply:
Regex question: I don't understand what the `[^]` at the beginning does, and https://regex101.com/ seems to have trouble parsing this regex.

Reply:
It’s one of the many ways to match _anything._ Your `(.|\n)` approach missed some characters (such as `\r`). Writing this I realize the `/s` (dotAll) flag has good support these days and we could have used that instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#browser_compatibility